src/api/app/views/webui2/webui/main/_add_status_message_modal.html.haml

```diff
@@ -0,0 +1,19 @@
+.modal#example-modal{:role => "dialog", :tabindex => "-1"}
```

Comment:
It would be nice to call ID which reflects what that modal does instead of example ;)